config/config.go

```diff
@@ -518,7 +518,9 @@ type CoProcessConfig struct {
 
 type CertificatesConfig struct {
 	API []string `json:"apis"`
-	// Specify upstream mutual TLS certificates at a global level in the following format: `{ "<host>": "<cert>" }``
+	// Specify upstream mutual TLS certificates at a global level as a map of domain -> certificate id or path
+	// In tyk.conf, this would be `"security": {"certificates": {"upstream": {"*": "/certs/default.pem", "example.com": "ab23ef123"}}},`; as an environmental variable it would be
```

Comment:
@andrei-tyk spotted that `environmental` should be `environment`

Reply:
Thanks @andrei-tyk @lghiur I have updated content to environment

Reply:
@lghiur could the PR be merged into main branch ?

Reply:
lgtm